releases/note-source/v17_0.rst

```diff
@@ -0,0 +1,299 @@
+For information about the technique adopted, refer to :dmtn:`037`; for documentation on the code, see `lsst.ip.diffim.DcrModel`.
+
+.. _release-v17-0-ppmb:
+
+The “Pessimistic Pattern Matcher b” is now the default astrometric matcher
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This new matcher works consistently over the large range of stellar densities which LSST will observe.
+The algorithm and associated validation testing are described in :dmtn:`031`; for more information, refer to `this community.lsst.org post`__.
+
+__ https://community.lsst.org/t/new-astrometric-matcher-is-now-the-default/3559
+
+.. _release-v17-0-gen3:
+
+“Generation 3” middleware
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release contains early versions of the “Generation 3” Data Butler and PipelineTask middleware which are being developed to satisify the requirements described in :ldm:`556`.
+This software is still under heavy development and is not currently deployed for regular data processing by the Data Management team; it should be regarded here as an unsupported preview.
+For more information, refer to the documentation for :ref:`lsst.pipe.base` and :ref:`lsst.daf.butler`.
+
+.. _release-v17-0-interface:
+
+Significant Interface Changes
+-----------------------------
+
+- :ref:`release-v17-0-runDataRef`
+- :ref:`release-v17-0-table-fields`
+- :ref:`release-v17-0-inverted`
+- :ref:`release-v17-0-calibration-flags`
+- :ref:`release-v17-0-blendedness-flags`
+- :ref:`release-v17-0-task-metadata`
+- :ref:`release-v17-0-pybind11`
+- :ref:`release-v17-0-astro-metadata-translator`
+- :ref:`release-v17-0-logging`
+
+.. _release-v17-0-runDataRef:
+
+Reworked Task entrypoints
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All ``Task``\s now provide a consistent interface to their primary functionality.
+The ``Task`` provides a ``run()`` method, which takes as explicit arguments everything the task needs to get its job done.
+``Task``\s may also provide a ``runDataRef()`` method, which takes a Butler “dataRef” as its argument, unpacks it, and calls ``run()``.
+Command line task runners call ``runDataRef()``.
+For further details, refer to `this community.lsst.org post`__ and :jira:`RFC-352`.
+
+__ https://community.lsst.org/t/api-change-for-tasks-rename-run-primarymethod-to-rundataref-run/3054
+
+.. _release-v17-0-table-fields:
+
+Updates to output table field names
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All table fields which previously had the suffix ``Sigma`` now have the suffix ``Err``.
+This reflects usage in the Data Products Definition Document (DPDD; :lse:`163`).
+For more information, refer to `this community.lsst.org post`__ and :jira:`RFC-333`.
+
+__ https://community.lsst.org/t/sigma-renamed-to-err-for-centroids-fluxes-and-aperture-corrections/3108
+
+Further, source catalogs have been updated to refer to to fluxes with the suffix ``instFlux`` (or “instrumental flux”), rather than simply ``flux`` (which is taken to refer a calibrated flux).
```

Comment:
Further, source catalogs have been updated to refer _(remove  duplicate 'to')_ to fluxes with the suffix ``instFlux`` (or “instrumental flux”), rather than simply ``flux`` (which is taken to refer **to** a calibrated flux).